lib/dotenv_util.rb

```diff
@@ -2,6 +2,14 @@
 
 # Provides Generic support for manipulating Dotenv files
 class DotenvUtil
+  ENV_LINE_PATTERN = /^ # A line
+    (?<key>[A-Z_]+) # Beginning with al alphanumeric key
+    =               # Then an equals sign
```

Comment:
should this be something like `=\s?` to accommodate for whitespace on the right side of the equal sign?

Reply:
Why would there be whitespace on the right side of the equal sign?

Reply:
because it's user input, and people do inconsistent shit.

```
WEBSITE_CLIENT_ID=5742
IPHONE_CLIENT_ID = 11589
ANDROID_CLIENT_ID=3631
OTHER_ANDROID_CLIENT_ID = 12049
```

Reply:
I was going to argue that we're slightly more responsible than users when it comes to configuration entry, but I found that [Dotenvs Parser](https://github.com/bkeepers/dotenv/blob/master/lib/dotenv/parser.rb#L14) supports such creative inputs, among others, so I admit that we should support the same.